### PR TITLE
Distinguish metrics by bot types

### PR DESCRIPTION
**Pull request by hweawer**
Distinguish metrics of different bots by adding a new env variable `PROMETHEUS_PREFIX` which can be one of:
- `depositor`
- `pauser`
- `unvetter`

### CODE REVIEW

File: src/metrics/metrics.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from prometheus_client.metrics import Counter, Gauge, Histogram
 
-PREFIX = 'depositor_bot'
+from variables import PROMETHEUS_PREFIX
+
+METRICS_PREFIX = PROMETHEUS_PREFIX + '_bot'
```
Comment (F4ever) [inline src/metrics/metrics.py:5]: This can be confusing, let's remove this prefix modification

File: src/metrics/metrics.py [hunk 2]
```diff
@@ -19,67 +21,69 @@
...
     'possible_deposits_amount',
     'Possible deposits amount.',
     ['module_id'],
-    namespace=PREFIX,
+    namespace=METRICS_PREFIX,
 )
 
-ETH_RPC_REQUESTS_DURATION = Histogram('eth_rpc_requests_duration', 'Duration of requests to ETH1 RPC', namespace=PREFIX)
+ETH_RPC_REQUESTS_DURATION = Histogram('eth_rpc_requests_duration', 'Duration of requests to ETH1 RPC',
+                                      namespace=METRICS_PREFIX)
```
Comment (F4ever) [inline src/metrics/metrics.py:79]: Fix formatting

File: src/variables.py [hunk 3]
```diff
@@ -72,6 +72,7 @@
 
 # Metrics
 PROMETHEUS_PORT = int(os.getenv('PROMETHEUS_PORT', '9000'))
+PROMETHEUS_PREFIX = os.getenv("PROMETHEUS_PREFIX", "depositor")
```
Comment (F4ever) [inline src/variables.py:75]: Use single quotes

Comment (F4ever) [reply src/variables.py:75]: This should be checked via ruff, but I didn't properly set up it in ci

File: src/metrics/metrics.py (hunk 1 above)
Comment (hweawer) [reply src/metrics/metrics.py]: Done

File: src/metrics/metrics.py (hunk 2 above)
Comment (hweawer) [reply src/metrics/metrics.py]: Done

**Review by F4ever — APPROVED** [review]

**Pull request merged**